### PR TITLE
#355: FEAT: Added support for additional options in EXPIRE and EXPIREAT.

### PR DESCRIPTION
**Pull request by raghavbabbar**
https://github.com/DiceDB/dice/issues/355

### CODE REVIEW

**Comment by raghavbabbar** [comment]
@JyotinderSingh  please review

File: core/eval.go [hunk 1]
```diff
@@ -652,22 +655,85 @@ func evalEXPIREAT(args []string, store *Store) []byte {
...
+			if prevExpiry == nil || *prevExpiry > newExpInMilli {
+				shouldSetExpiry = false
+			}
+		case constants.LT:
+			ltCmd = true
+			if prevExpiry != nil && *prevExpiry < newExpInMilli {
+				shouldSetExpiry = false
+			}
+		default:
+			return false, errors.New("ERR Unsupported option " + subCommands[i])
```
Comment (JyotinderSingh) [inline core/eval.go:727]: Please use the error builder utilities to form the error messages. You can refer to other such usages in the code for guidance.
Same for other instances.

**Review by JyotinderSingh — CHANGES_REQUESTED** [review]
Thanks a lot for adding support for the additional options @raghavbabbar! The changes look good, just had a minor comment.

File: core/eval.go (hunk 1 above)
Comment (raghavbabbar) [reply core/eval.go]: Done

**Review by JyotinderSingh — APPROVED** [review]

**Pull request merged**